.github/workflows/continuous-deployment.yml

```diff
@@ -165,7 +164,10 @@ jobs:
 
           java -jar pipeline-scan.jar \
           -vid ${{ secrets.VERACODE_API_ID }} -vkey ${{ secrets.VERACODE_API_KEY }} \
-          -bf results-latest.json -jf results.json -fjf filtered_results.json \
+          -bf results-latest.json \
+          -jf results.json \
+          -fjf filtered_results.json \
+          --fail_on_severity="Very High, High, Medium" \
```

Comment:
This line might be a problem. Seeing a parsing issue with `Very High` that last time we had this enabled.

Reply:
@magile32 removed that line